[2024-12-17T14:49:31.251+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T14:49:31.260+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 scheduled__2024-12-16T00:00:00+00:00 [queued]>
[2024-12-17T14:49:31.266+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 scheduled__2024-12-16T00:00:00+00:00 [queued]>
[2024-12-17T14:49:31.266+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-17T14:49:31.272+0000] {taskinstance.py:2889} INFO - Executing <Task(TriggerDagRunOperator): trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2> on 2024-12-16 00:00:00+00:00
[2024-12-17T14:49:31.274+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=278) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-17T14:49:31.274+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'AF_TEST1', 'trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2', 'scheduled__2024-12-16T00:00:00+00:00', '--job-id', '2036', '--raw', '--subdir', 'DAGS_FOLDER/script/AF_TEST1.py', '--cfg-path', '/tmp/tmp_9yt8oq9']
[2024-12-17T14:49:31.275+0000] {standard_task_runner.py:72} INFO - Started process 282 to run task
[2024-12-17T14:49:31.275+0000] {standard_task_runner.py:105} INFO - Job 2036: Subtask trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2
[2024-12-17T14:49:31.295+0000] {task_command.py:467} INFO - Running <TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 scheduled__2024-12-16T00:00:00+00:00 [running]> on host 1e4be155f297
[2024-12-17T14:49:31.322+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test-framework' AIRFLOW_CTX_DAG_ID='AF_TEST1' AIRFLOW_CTX_TASK_ID='trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-16T00:00:00+00:00'
[2024-12-17T14:49:31.323+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T14:49:33.160+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:175 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.160+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:222 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.161+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1151 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.162+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1248 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.162+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1271 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.163+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1406 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.163+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1439 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.163+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1495 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.164+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1683 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.164+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1750 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.164+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2025 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.165+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2110 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.165+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2407 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.165+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2468 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.167+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:3987 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.168+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4096 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.168+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4127 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.168+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4186 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.168+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4225 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.169+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4275 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.170+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5209 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.170+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5265 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.170+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5302 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.171+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5358 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.172+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6437 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.172+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6497 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.172+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6538 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.173+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.173+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7023 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.174+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7096 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.174+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7267 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.174+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7333 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.174+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7378 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.175+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7440 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.176+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8222 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.176+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8282 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.176+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8322 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.176+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8368 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.178+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:9475 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.178+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:9531 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.179+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.180+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.180+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10625 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.180+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10680 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.181+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10923 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.181+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10979 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.181+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11016 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.182+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11072 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.182+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11231 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.182+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11282 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.183+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11420 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.183+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11477 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.184+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12300 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.184+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12383 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.184+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12439 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.185+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12524 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.185+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13019 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.186+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13090 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.186+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13139 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.187+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13200 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.190+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13886 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.191+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13941 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.191+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13977 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.192+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14026 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.192+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14318 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.193+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14388 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.193+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14436 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.193+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14492 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.194+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14744 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.194+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14810 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.195+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15415 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.195+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15482 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.196+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15527 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.196+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15578 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.197+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15897 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.197+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15953 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.198+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16617 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.198+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16674 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.198+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16712 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.199+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16780 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.199+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.199+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16988 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.200+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17025 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.200+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17090 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.200+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17228 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.201+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17284 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.204+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20234 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.204+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20289 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.205+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20862 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.205+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20921 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.206+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20960 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.206+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21014 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.208+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21316 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.208+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21377 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.208+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21418 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.209+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21478 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.209+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21612 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.209+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21668 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.210+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22155 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.210+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22218 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.211+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22559 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.211+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22626 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.211+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22671 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.212+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22722 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.212+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23061 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.213+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23128 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.213+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23173 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.213+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23224 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.214+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23511 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.214+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23567 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.214+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23968 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.215+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:24018 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.216+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25343 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.217+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25398 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.217+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25434 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.217+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25485 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.219+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:26723 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.219+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:26779 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.219+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27121 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.220+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27177 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.220+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27461 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.220+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27517 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.221+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28053 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.221+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28109 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.222+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28536 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.222+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28592 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.223+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.223+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28982 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.225+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:30727 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.225+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:30783 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.226+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31296 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.226+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31344 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.227+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31617 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.227+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31673 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.228+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32120 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.228+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32176 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.228+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32339 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.229+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32395 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.229+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32432 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.229+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32487 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.230+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33502 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.231+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33558 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.231+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33869 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.232+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33928 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.232+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33967 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.232+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34021 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.233+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34739 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.233+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34806 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.234+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34852 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.234+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34908 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.236+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:36915 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.236+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:36971 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.237+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37417 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.237+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37473 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.238+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37845 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.238+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37901 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.239+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38208 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.239+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38264 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.239+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38301 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.240+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38351 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.240+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38690 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.241+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38758 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.241+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39141 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.241+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39192 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.242+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39419 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.242+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39487 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.243+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39683 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.243+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39738 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.243+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:40019 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.244+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:40084 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.245+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41138 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.245+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41196 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.246+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41359 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.246+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41415 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.246+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41452 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.246+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41507 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.247+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41875 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.247+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41947 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.251+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42441 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.251+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42497 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.251+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42534 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.251+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42590 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.252+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42627 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.252+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42692 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.252+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43020 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.253+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43079 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.253+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43118 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.253+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43168 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.254+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43514 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.254+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43573 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.254+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43612 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.255+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43666 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.255+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43989 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.256+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44045 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.256+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44386 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.256+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44442 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.258+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45003 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.258+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45058 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.259+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45444 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.259+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.260+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46042 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.260+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46098 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.261+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46702 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.261+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46758 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.262+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47000 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.262+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47050 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.263+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47648 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.263+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47704 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.264+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48113 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.264+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48190 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.264+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48244 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.265+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48321 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.266+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.266+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49555 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.267+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.267+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50002 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.267+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50052 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.268+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50107 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.268+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50379 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.268+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50449 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.269+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.269+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50554 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.269+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50590 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.270+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.270+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50710 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.270+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50770 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.271+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51046 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.271+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51116 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.271+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51166 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.271+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51226 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.272+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51485 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.272+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51540 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.273+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51787 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.273+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51851 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.273+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51895 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.273+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51958 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.274+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52212 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.274+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52275 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.275+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52463 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.275+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.276+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53040 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.276+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53111 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.276+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53334 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.277+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53407 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.277+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53961 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.278+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54059 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.278+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54322 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.278+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54398 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.279+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55022 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.280+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55078 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.280+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55310 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.280+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55365 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.281+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56196 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.282+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56248 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.282+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56508 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.282+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56564 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.283+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57087 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.283+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57147 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.284+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57187 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.284+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57237 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.284+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57560 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.285+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57620 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.285+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.285+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57710 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.287+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59106 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.287+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59162 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.288+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59316 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.288+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59396 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.288+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59452 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.289+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59535 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.289+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59594 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.289+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59677 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.289+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59733 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.290+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59818 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.291+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60639 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.291+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60722 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.291+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60778 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.292+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60863 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.292+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61167 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.293+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61250 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.293+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61306 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.293+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61387 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.294+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61696 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.294+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61752 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.294+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61954 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.295+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61992 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.296+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:62660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.296+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:62715 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.297+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:63882 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.297+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:63946 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.298+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64582 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.299+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64674 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.299+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64940 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.300+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65014 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.300+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65117 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.300+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65199 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.301+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65385 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.301+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65441 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.302+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65874 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.302+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65950 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.302+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66003 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.303+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66087 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.303+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66686 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.304+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66779 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.304+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66918 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.304+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66964 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.305+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:67735 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.306+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:67791 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.306+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68043 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.306+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68097 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.307+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.307+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T14:49:33.556+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T14:49:33.556+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=AF_TEST1, task_id=trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2, run_id=scheduled__2024-12-16T00:00:00+00:00, execution_date=20241216T000000, start_date=20241217T144931, end_date=20241217T144933
[2024-12-17T14:49:33.571+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T14:49:33.579+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-17T14:49:33.580+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
